src/components/CaptureFilter.js

```diff
@@ -76,6 +77,7 @@ const styles = (theme) => {
 
 function Filter(props) {
   const speciesContext = useContext(SpeciesContext);
+  const { isLoading } = useContext(SpeciesContext);
```

Comment:
Just use `speciesContext.isLoading`. Its meaning is clearer.

Reply:
acknowledged and updated